Homework_5.ipynb

```diff
@@ -0,0 +1,651 @@
+{
```

Comment:
<p>This is a good way to figure out the size on your platform. However the size of double really is misleading here since you will loose precision for that particular series.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_PythonCpp/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>